windows/progress-view/progress-view.vcxproj

```diff
@@ -162,18 +161,25 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ReactNativeWindowsTargets">
```

Comment:
Adding the import of the CppLib.targets is correct, but you also need to add the import for the CppLib.props earlier in the file, and remove the project references for Microsoft.ReactNative and Microsoft.ReactNative.Cxx.

Reply:
You can see similar changes I made for react-native-video's project here: https://github.com/react-native-video/react-native-video/pull/2206/files#diff-21a98c9b66b676799e297b1bf6028f32087df1baf8ebbc109181f75e8348a4e6

Reply:
Thanks for the feedback, I'll update the PR tomorrow.